Tests/StencilSwiftKitTests/CallNodeTests.swift

```diff
@@ -38,7 +38,8 @@ class CallNodeTests: XCTestCase {
     }
 
     XCTAssertEqual(node.variableName, "myFunc")
-    XCTAssertEqual(node.arguments, [Variable("a"), Variable("b"), Variable("c")])
+    let variables = node.arguments.flatMap({ $0 as? FilterExpression }).flatMap({ $0.variable })
```

Comment:
I generally agree with this code style too when chaining flatMaps but our linter disagrees there, so maybe adapt the syntax here, like `let variables = node.arguments.flatMap { ($0 as? FilterExpression)?.variable }`?